gui/src/containers/JobSpec.js

```diff
@@ -84,13 +92,20 @@ const renderJobSpec = ({classes, jobSpec, latestJobRuns, jobRunsCount}) => (
   </Grid>
 )
 
-const renderLatestRuns = ({classes, latestJobRuns}) => (
+const renderLatestRuns = ({jobSpecId, classes, latestJobRuns, jobRunsCount}) => (
   <React.Fragment>
     <Typography variant='title' className={classes.lastRun}>
       Last Run
     </Typography>
 
-    <JobRunsList runs={latestJobRuns} />
+    <Card>
+      <JobRunsList runs={latestJobRuns} />
+    </Card>
+    {jobRunsCount > LATEST_JOB_RUNS_COUNT &&
```

Comment:
Is there not an `{if ...}markup{end}` syntax we could use here?

Reply:
This is how you do inline `if` in JSX https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

The other option is to pull it out into a variable but I prefer this style.